dynamiqs/utils/wigner.py

```diff
@@ -19,16 +19,18 @@ def wigner(
     n_pixels: int = 200,
     method: Literal['clenshaw', 'fft'] = 'clenshaw',
 ) -> tuple[Tensor, Tensor, Tensor]:
-    """Compute the wigner distribution of a state vector or density matrix.
+    """Compute the wigner distribution of a ket or density matrix.
 
     Args:
-        state: State vector or density matrix.
-        x_max: Maximum value of x for which to compute the wigner distribution.
-        p_max: Maximum value of p for which to compute the wigner distribution.
+        state (..., n, 1) or (..., n, n): State vector or density matrix.
+        x_max (float): Maximum value of x for which to compute the wigner distribution.
+            Defaults to `2 * pi`.
```

Comment:
This feels very weird to me, the wigner values for each pixel range between -2/pi and 2/pi, but why do we default the Wigner border to such values? It should be a coherent state amplitude IMO, e.g. `6.0`.

Reply:
This is because of the `fft` method. When computing the wigner through the fft, the range in P is related to the range in X through a Fourier transform. So if you want a square range, the range in both directions should be [-2pi, 2pi]. Otherwise one direction is squeezed with respect to the other.

For the default method `clenshaw`, this is however not required. One can compute the Wigner in any desired range.